File: src/main/kotlin/fr/jhelp/kotlinLightSamples/CompanionTest.kt
package fr.jhelp.kotlinLightSamples

class CompanionTest
{
    companion object
    {
        var before = "before"

        /**
         * Wonderful world }}}}
         */
        fun staticMethod()
        {
            val k = '}'
            println("static {{{{{ $k")
        }

        // ploki { {{{{

        val after = "after"
    }

    fun instanceMethod()
    {
        val l = '\n'
        val m = '\u00af'
        println("instance $l hh $m pp")
    }
}

File: src/main/swift/Sources/swift/fr/jhelp/kotlinLightSamples/CompanionTest.swift

class CompanionTest {

    
        static var before = "before"

        /**
         * Wonderful world }}}}
         */
        static func staticMethod()
        {
            let k = "}"
            print("static {{{{{ \(k)")
        }

        // ploki { {{{{

        static let after = "after"
    

    func instanceMethod()
    {
        let l = "\n"
        let m = "\u{00af}"
        print("instance \(l) hh \(m) pp")
    }
}
